app/Http/Controllers/Api/AuthController.php

```diff
@@ -33,20 +34,30 @@ public function uuid()
      */
     public function login(Request $request)
     {
+        $validator = Validator::make($request->all(), [
+            'email' => 'required',
```

Comment:
Please use username and password instead. Username = email || 'Matric Number' depending on who is login into the system.